connectors/elasticsearch-sink/3.0.1.4/elasticsearch-sink.md

```diff
@@ -0,0 +1,161 @@
+---
+description: The Elasticsearch sink connector pulls messages from Pulsar topics and persists the messages to indexes
+author: ["ASF"]
+contributors: ["ASF"]
+language: Java
+document: 
+source: "https://github.com/streamnative/pulsar/tree/v3.0.1.4/pulsar-io/elastic-search"
+license: Apache License 2.0
+tags: ["Pulsar IO", "Elasticsearch", "Sink"]
+alias: Elasticsearch Sink Connector
+features: ["Use Elasticsearch sink connector to sync data from Pulsar"]
+license_link: "https://www.apache.org/licenses/LICENSE-2.0"
+icon: "/images/connectors/elasticsearch.png"
+download: "https://github.com/streamnative/pulsar/releases/download/v3.0.1.4/pulsar-io-elastic-search-3.0.1.4.nar"
+support: StreamNative
+support_link: https://streamnative.io
+support_img: "/images/streamnative.png"
+owner_name: ""
+owner_img: ""
+dockerfile: https://hub.docker.com/r/streamnative/pulsar-io-elastic-search
+sn_available: true
+id: "elasticsearch-sink"
+---
+
+The [Elasticsearch](https://www.elastic.co/elasticsearch/) sink connector pulls messages from Pulsar topics and persists the messages to indexes. For more information about connectors, see [Connector Overview](https://docs.streamnative.io/docs/connector-overview).
+
+![](/images/connectors/elasticsearch-sink.png)
+
+This document introduces how to get started with creating an Elasticsearch sink connector and get it up and running.
+## Quick start
+
+### Prerequisites
+
+The prerequisites for connecting an Elasticsearch sink connector to external systems include:
```

Comment:
What versions do we support? I think need to add this information. You can refer to:

https://github.com/apache/pulsar/blob/b7eab9469177eda2c56e36bb9871aab48a17d4ec/pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchConfig.java#L273-L282